pkg/deploy/deploy_test.go

```diff
@@ -95,8 +95,8 @@ func TestDeployConfigGraph_SingleConfig(t *testing.T) {
 		},
 	}
 
-	dummyClient := dtclient.DummyClient{}
-	clientSet := &client.ClientSet{ClassicClient: &dummyClient, SettingsClient: &dummyClient}
+	dummyClient := dtclient.DummyConfigClient{}
+	clientSet := &client.ClientSet{ConfigClient: &dummyClient, SettingsClient: &dtclient.DummySettingsClient{}}
```

Comment:
Does it make sense to pass both parameters in the same way, or is the `dummyClient` needed in a variable as it's used later on?

Reply:
dummyClient is being used later, as the classic dummy client records entries when requests are made. But this is something that should be re-examined and made uniform for all dummy clients.